CHANGELOG

```diff
@@ -1,30 +1,74 @@
 cocos2d-x-3.10 December ? 2015
-[NEW]           Core:  Added Application::getVersion() to get the app version.
+
+[HIGHLIGHT]     UI: Rewrite Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
+[HIGHLIGHT]     UI: Change PageView to derive from ListView.
+
+[NEW]           Core: Added Application::getVersion() to get the app version.
 [NEW]           UI: Add PageView indicator.
-[NEW]           Label: Add three Overflow type to new label, see release note for more information.
+[NEW]           UI: Label add three Overflow type to new label, see release note for more information.
 [NEW]           UI: UIText::clone supports clone the text effect.
 [NEW]           Label: Add methods to query label effect state.
+[NEW]           UI: UIRichText support the new line element.
+[NEW]           UI: Scale9Sprite fade actions with cascade opacity.
+[NEW]           UI: Pageview support adjust child size.
```

Comment:
This one `+[NEW]           UI: Pageview support adjust child size.` is not needed.
It's a part of `[HIGHLIGHT]     UI: Change PageView to derive from ListView.`